app/passport/views.py

```diff
@@ -103,3 +105,80 @@ def passport(request, pattern):
         ],
     }
     return JsonResponse(context)
+
+
+def verifiable_credential(request):
```

Comment:
let's use the login decorator that django offers 
@login_required

Reply:
Resolved in https://github.com/gitcoinco/web/pull/9032/commits/91afdb82d0b077c0955311ec642a210ea34ab0af